bin/makeMarkerStats.R

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+
+option_list = list(
+  make_option(
+    c("-c", "--counts-dir"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "Path to old-school 10X dir with barcodes.tsv, genes.tsv and matrix.mtx, for count data"
+  ),
+  make_option(
+    c("-j", "--clusters-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "File containing cluster definitions as downloaded from a Scanpy workflow with scanpy-scripts"
+  ),
+  make_option(
+    c("-d", "--cluster-markers-dir"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "Directory path in which to find marker files as downloaded from a Scanpy workflow with scanpy-scripts"
+  ),
+  make_option(
+    c("-m", "--cluster-markers-file-pattern"),
+    action = "store",
+    default = "markers_[0-9]*.tsv",
+    type = 'character',
+    help = "A pattern used to match marker files as downloaded from a Scanpy workflow with scanpy-scripts"
+  ),
+  make_option(
+    c("-g", "--cellgroups-file"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "A tabular file in which additional cell groups column can be found"
+  ),
+  make_option(
```

Comment:
I wonder if this model of treating clusters and celltypes explicitly differently might not give us problems in the future. Shouldn't we maybe simply process groups of cells, regardless of their provenance? We could this leave for later do, but I envision doing this with diseases for instance very soon.

Reply:
This isn't too bad- I've only separated the unsupervised clustering from the 'annotation based' clustering. The only edit required will be to replace the 'celltype-markers-file' argument with some parameter listing a variety of files on various cell groups. 

But yes, we could also integrate annotation and clustering groupings.